### PR TITLE
chore(rating): update to .42

### PR DESCRIPTION
**Pull request by Foxandxss**
[Plunker](http://plnkr.co/edit/2wtcZ5bmQk8xmvwWvFP6?p=preview)

### CODE REVIEW

File: src/components/rating.ts [hunk 1]
```diff
@@ -1,11 +1,9 @@
-import {Component, View, NgFor, NgClass, EventEmitter, OnInit} from 'angular2/angular2';
+import {Component, NgFor, NgClass, EventEmitter, OnInit} from 'angular2/angular2';
 
 @Component({
   selector: 'ngb-rating',
   inputs: ['max', 'rate', 'readonly'],
-  outputs: ['updateRating: rate', 'onHover', 'onLeave']
-})
-@View({
+  outputs: ['updateRating: rate', 'onHover', 'onLeave'],
```
Comment (pkozlowski-opensource) [inline src/components/rating.ts]: Maybe we could use `@Input` and `@Output` decorators instead?

Comment (Foxandxss) [inline src/components/rating.ts]: I am not familiar with that. Will take a peek.

Comment (pkozlowski-opensource) [inline src/components/rating.ts]: This is just a decorator that you stick on a class member (property) to mark it as input / output.

Comment (Foxandxss) [inline src/components/rating.ts]: Yeah, saw it. Will update later.

**Comment by Foxandxss** [comment]
Alright @pkozlowski-opensource I updated all three PRs with those changes. Also updated here the new desugar of two-way databinding.

**Comment by wesleycho** [comment]
LGTM

**Pull request closed without merging**